From 05bfc93c187ed81820820d9667b215974710a309 Mon Sep 17 00:00:00 2001
From: John Galt <johngaltfirstrun@gmail.com>
Date: Tue, 30 Apr 2024 10:32:19 -0400
Subject: [PATCH] cc/lto: instr limit of 40 for explicit LTO modules

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 cc/lto.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cc/lto.go b/cc/lto.go
index 44441527a..984ada3e6 100644
--- a/cc/lto.go
+++ b/cc/lto.go
@@ -134,7 +134,7 @@ func (lto *lto) flags(ctx ModuleContext, flags Flags) Flags {
 		// Reduce the inlining threshold for a better balance of binary size and
 		// performance.
 		if !ctx.Darwin() {
-			if ctx.isAfdoCompile(ctx) {
+			if ctx.isAfdoCompile(ctx) || lto.ThinLTO() {
 				ltoLdFlags = append(ltoLdFlags, "-Wl,-plugin-opt,-import-instr-limit=40")
 			} else {
 				ltoLdFlags = append(ltoLdFlags, "-Wl,-plugin-opt,-import-instr-limit=5")
-- 
2.47.1.windows.1

