From 87ea48ccb6b7e2a73a1b9878d3d9963b34cf86f5 Mon Sep 17 00:00:00 2001
From: Igor <sorocean.igor@gmail.com>
Date: Mon, 3 Nov 2025 08:17:19 +0200
Subject: [PATCH] make it privileged

---
 app/Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/app/Android.bp b/app/Android.bp
index 3f778d6..29fc7bb 100644
--- a/app/Android.bp
+++ b/app/Android.bp
@@ -24,5 +24,6 @@ android_app {
         enabled: false,
     },
     platform_apis: true,
+    privileged: true,
     system_ext_specific: true,
 }
-- 
2.51.2.windows.1

