project bionic/
diff --git a/libc/Android.bp b/libc/Android.bp
index efba83b..d058040 100644
--- a/libc/Android.bp
+++ b/libc/Android.bp
@@ -173,8 +173,10 @@ cc_defaults {
 
     whole_static_libs: [
         "libscudo",
+        "libmimalloc",
     ],
     cflags: [
+        "-DUSE_MIMALLOC",
         "-DUSE_SCUDO",
     ],
     header_libs: ["gwp_asan_headers"],
@@ -187,6 +189,7 @@ cc_defaults {
             ],
             exclude_static_libs: [
                 "libscudo",
+                "libmimalloc",
             ],
         },
     },
@@ -1738,6 +1741,7 @@ cc_library_headers {
         "//external/scudo",
         "//system/core/property_service/libpropertyinfoparser",
         "//system/extras/toolchain-extras",
+        "//external/mimalloc",
     ],
 
     stl: "none",
diff --git a/libc/bionic/malloc_common.h b/libc/bionic/malloc_common.h
index 4afcc4a..4294b82 100644
--- a/libc/bionic/malloc_common.h
+++ b/libc/bionic/malloc_common.h
@@ -55,7 +55,12 @@ __END_DECLS
 
 #else // __has_feature(hwaddress_sanitizer)
 
-#if defined(USE_SCUDO)
+#if defined(USE_MIMALLOC)
+
+#include <mimalloc_adapt.h>
+#define Malloc(function)  mimalloc_ ## function
+
+#elif defined(USE_SCUDO)
 
 #include "scudo.h"
 #define Malloc(function)  scudo_ ## function

