From 0e4dc0806b4c5b576a822be36b9717ebd242fb15 Mon Sep 17 00:00:00 2001
From: Pranav Vashi <neobuddy89@gmail.com>
Date: Sun, 31 Aug 2025 09:31:38 +0530
Subject: [PATCH] Settings: Hide development options completely when disabled

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 .../system/DeveloperOptionsController.kt         | 16 +++++++++++++---
 1 file changed, 13 insertions(+), 3 deletions(-)

diff --git a/src/com/android/settings/system/DeveloperOptionsController.kt b/src/com/android/settings/system/DeveloperOptionsController.kt
index 6874ec7552a..d4d1ca5a12b 100644
--- a/src/com/android/settings/system/DeveloperOptionsController.kt
+++ b/src/com/android/settings/system/DeveloperOptionsController.kt
@@ -42,9 +42,19 @@ import com.android.settingslib.spaprivileged.template.preference.RestrictedPrefe
 class DeveloperOptionsController(context: Context, preferenceKey: String) :
     ComposePreferenceController(context, preferenceKey) {
 
-    override fun getAvailabilityStatus() =
-        if (mContext.userManager.isAdminUser) AVAILABLE
-        else DISABLED_FOR_USER
+    override fun getAvailabilityStatus(): Int {
+        if (!isDevelopmentSettingsEnabled()) return CONDITIONALLY_UNAVAILABLE
+        return if (mContext.userManager.isAdminUser) AVAILABLE else DISABLED_FOR_USER
+    }
+
+    private fun isDevelopmentSettingsEnabled(): Boolean {
+        val def = if (Build.IS_ENG) 1 else 0
+        return Settings.Global.getInt(
+            mContext.contentResolver,
+            Settings.Global.DEVELOPMENT_SETTINGS_ENABLED,
+            def
+        ) != 0
+    }
 
     private val isDevelopmentSettingsEnabledFlow = context.settingsGlobalBooleanFlow(
         name = Settings.Global.DEVELOPMENT_SETTINGS_ENABLED,
-- 
2.47.1.windows.1

