From 745fda3a380a2c923c497294d33f22d0b7903497 Mon Sep 17 00:00:00 2001
From: Pranav Vashi <neobuddy89@gmail.com>
Date: Sat, 5 Jul 2025 19:49:58 +0530
Subject: [PATCH] SettingsLib: Prevent NPE for fragments without divider

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 .../settingslib/widget/SettingsBasePreferenceFragment.kt        | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SettingsLib/SettingsTheme/src/com/android/settingslib/widget/SettingsBasePreferenceFragment.kt b/packages/SettingsLib/SettingsTheme/src/com/android/settingslib/widget/SettingsBasePreferenceFragment.kt
index 8d12f01e24ed..7956b385f570 100644
--- a/packages/SettingsLib/SettingsTheme/src/com/android/settingslib/widget/SettingsBasePreferenceFragment.kt
+++ b/packages/SettingsLib/SettingsTheme/src/com/android/settingslib/widget/SettingsBasePreferenceFragment.kt
@@ -40,7 +40,7 @@ abstract class SettingsBasePreferenceFragment : PreferenceFragmentCompat() {
     @CallSuper
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        if (SettingsThemeHelper.isExpressiveTheme(requireContext())) {
+        if (SettingsThemeHelper.isExpressiveTheme(requireContext()) && listView != null) {
             // Don't allow any divider in between the preferences in expressive design.
             setDivider(null)
         }
-- 
2.51.2.windows.1

