From 8b5234e69add5eb755f72c89d278078a9cd32867 Mon Sep 17 00:00:00 2001
From: Pranav Vashi <neobuddy89@gmail.com>
Date: Tue, 10 Feb 2026 04:48:00 +0800
Subject: [PATCH] gen_build_prop: Do not leak lineage_ prefixes in build.prop

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 scripts/gen_build_prop.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/gen_build_prop.py b/scripts/gen_build_prop.py
index aa7210996..d6f0dcada 100644
--- a/scripts/gen_build_prop.py
+++ b/scripts/gen_build_prop.py
@@ -34,7 +34,7 @@ def get_build_variant(product_config):
     return "user"
 
 def get_build_flavor(product_config):
-  build_flavor = product_config["DeviceProduct"] + "-" + get_build_variant(product_config)
+  build_flavor = product_config["DeviceName"] + "-" + get_build_variant(product_config)
   if "address" in product_config.get("SanitizeDevice", []) and "_asan" not in build_flavor:
     build_flavor += "_asan"
   return build_flavor
@@ -118,7 +118,7 @@ def parse_args():
   # build_desc is human readable strings that describe this build. This has the same info as the
   # build fingerprint.
   # e.g. "aosp_cf_x86_64_phone-userdebug VanillaIceCream MAIN eng.20240319.143939 test-keys"
-  config["BuildDesc"] = f"{config['DeviceProduct']}-{config['BuildVariant']} " \
+  config["BuildDesc"] = f"{config['DeviceName']}-{config['BuildVariant']} " \
                         f"{config['Platform_version_name']} {config['BuildId']} " \
                         f"{config['BuildNumber']} {config['BuildVersionTags']}"
 
@@ -155,7 +155,7 @@ def generate_common_build_props(args):
   print(f"ro.product.{partition}.device={config['DeviceName']}")
   print(f"ro.product.{partition}.manufacturer={config['ProductManufacturer']}")
   print(f"ro.product.{partition}.model={config['ProductModel']}")
-  print(f"ro.product.{partition}.name={config['DeviceProduct']}")
+  print(f"ro.product.{partition}.name={config['DeviceName']}")
 
   if partition != "system":
     if config["ProductModelForAttestation"]:
-- 
2.50.1.windows.1

