From 368210ac48437cc166e7f5553ff0e145adfe8895 Mon Sep 17 00:00:00 2001
From: Pranav Vashi <neobuddy89@gmail.com>
Date: Thu, 8 Sep 2022 02:34:08 +0530
Subject: [PATCH] soong: Add kryo300 variant support

Signed-off-by: Pranav Vashi <neobuddy89@gmail.com>
---
 android/arch_list.go | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/android/arch_list.go b/android/arch_list.go
index 081c286f3..3974dd1e8 100644
--- a/android/arch_list.go
+++ b/android/arch_list.go
@@ -92,6 +92,7 @@ var cpuVariants = map[ArchType][]string{
 		"cortex-a76",
 		"krait",
 		"kryo",
+		"kryo300",
 		"kryo385",
 		"exynos-m1",
 		"exynos-m2",
@@ -105,6 +106,7 @@ var cpuVariants = map[ArchType][]string{
 		"cortex-a75",
 		"cortex-a76",
 		"kryo",
+		"kryo300",
 		"kryo385",
 		"kryo785",
 		"exynos-m1",
-- 
2.47.1.windows.1

